### PR TITLE
Bookstore: create and remove books

### PR DESCRIPTION
**Pull request by rachidelaid**
> tasks done;
- installed `redux-logger`.
- configured the store using only the books reducer and `redux-logger` as a middleware.
- wrapped the whole `App` around the redux provider.
- defined books action types.
- added `addBook` and `removeBook` action creators.
- added the book reducer and exported it as the default.
- imported `useDispatch` from both `BookCard` and `Form` components.
- imported `addBook` in the `Form`.
- imported `removeBook` in the `BookCard`.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
```
Comment (hiromataba) [inline src/redux/books/books.js:9]: - [ ] For now, I am trying to add a book but cannot see it on the UI please  try to fix that and display the books on the `UI` also remove to add the possibility to remove a book

https://user-images.githubusercontent.com/75126481/155113289-7e503fe0-4ac6-4114-a7af-9e37965a06dd.MP4

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @rachidelaid :wave: :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To highlight :champagne: 
- Nice PR title and description :heavy_check_mark: 
- Professional README file :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js [hunk 2]
```diff
@@ -1,23 +1,55 @@
...
+      <form>
+        <input
+          type="text"
+          placeholder="Book title"
+          required
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <select value={category} onChange={(e) => setCategory(e.target.value)}>
+          <option value="" disabled selected>
```
Comment (TedLivist) [inline src/components/Form.js:40]: - [ ] **[OPTIONAL]** I would recommend you resolve this warning in your console to keep with conventional code.

  rachid-select

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @rachidelaid,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**